proto/zitadel/admin.proto

```diff
@@ -4061,6 +4065,22 @@ message DataOIDCApplication {
     string app_id = 1;
     zitadel.management.v1.AddOIDCAppRequest app = 2;
 }
+message DataAppKey {
+    string id = 1;
+    string project_id = 2;
+    string app_id = 3;
+    string client_id = 4;
```

Comment:
what is that needed for?

Reply:
"that"? do you mean the clientID? if so, to create the authnkey the clientID is used as identifier

Reply:
ok we should then chang the command / writemodel as this is a projection issue (doesn't have to be right now)